### PR TITLE
+Add create API object popup

### PR DESCRIPTION
**Pull request by wongpratan**

### CODE REVIEW

File: src/rootPages/Designer/ui_work_object_list_newObject_api.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                        },
+                        {
+                           view: "button",
+                           id: this.ids.buttonSave,
+                           css: "webix_primary",
+                           value: L("Save"),
+                           autowidth: true,
+                           type: "form",
+                           click: () => {
+                              return this.save();
```
Comment (nh758) [inline src/rootPages/Designer/ui_work_object_list_newObject_api.js:94]: I'm wondering if we should disable the save until we have more functionality built? Or add some warning that it's not fully functional.

**Review by nh758 — APPROVED** [review]
Good start. The only question is about how to communicate to users that it's not fully working yet.

File: src/rootPages/Designer/ui_work_object_list_newObject_api.js (hunk 1 above)
Comment (wongpratan) [reply src/rootPages/Designer/ui_work_object_list_newObject_api.js:94]: Yes, We will not merge this merge until all of task series will be done.

**Review by guyyoo — APPROVED** [review]
LGTM.

**Pull request merged**